Gopkg.toml

```diff
@@ -49,6 +50,11 @@ required = [
   name = "github.com/bsm/sarama-cluster"
   version = "=2.1.13"
 
+[[constraint]]
+  name = "github.com/russross/blackfriday"
+  version = "=1.5"
```

Comment:
Why do we need this as explicit dependency, instead of transitive via cobra?

Reply:
I could not make it work that way